modules/fuzztest/metadata.json

```diff
@@ -0,0 +1,17 @@
+{
+    "homepage": "https://github.com/google/fuzztest",
+    "maintainers": [
+        {
+            "email": "lszekeres@google.com",
+            "github": "lszekeres",
```

Comment:
@lszekeres - Is it okay that I added you as a maintainer of this module?

Reply:
Yes, thanks!